src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -42,6 +43,8 @@ public static String getEditPersonDescriptorDetails(EditPersonDescriptor descrip
         descriptor.getPhone().ifPresent(phone -> sb.append(PREFIX_PHONE).append(phone.value).append(" "));
         descriptor.getEmail().ifPresent(email -> sb.append(PREFIX_EMAIL).append(email.value).append(" "));
         descriptor.getAddress().ifPresent(address -> sb.append(PREFIX_ADDRESS).append(address.value).append(" "));
+        descriptor.getAdditionalNotes().ifPresent(additionalNotes -> sb
+                .append(PREFIX_ADDITIONAL_NOTES).append(additionalNotes.notes).append(" "));
```

Comment:
when separating a line, we should [prefer higher-level breaks to lower-level breaks.](https://se-education.org/guides/conventions/java/index.html) and in this case I think it would be better if we split by the end of `ifPresent(`